### PR TITLE
Workflow Automated Test - LRQA-68849 | master

### PR DESCRIPTION
**Pull request by evanilson-santana**

### CODE REVIEW

**Comment by evanilson-santana** [comment]
ci:test:sf

**Comment by evanilson-santana** [comment]
ci:test:sf

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase [hunk 1]
```diff
@@ -315,12 +315,45 @@ definition {
...
+			workflowTask = "Review");
 
-		// TODO LRQA-68849 CanOrderAssignedtoMeTasksByDueDate pending implementation
+		Notifications.goToFilterValues(filterValues = "Due Date");
 
+		AssertTextEquals.assertPartialText(
+			key_workflowAssetTitle = "Blogs Entry",
+			key_workflowTask = "Review",
+			locator1 = "MyWorkflowTasks#ASSIGNED_TO_MY_ROLES_TABLE_DUE_DATE",
+			value1 = "Within 8 Months");
```
Comment (lucasperj) [inline modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase:356]: Try to create macros from these functions when you feel the need to use them in .testcase files. Complementing the comment above, ideal is to use a macro to ensure the position of items, you can replace this piece and use the same macro viewMyWorkflowTaskItemPosition and path MY_WORKFLOW_TASK_ITEM_POSITION that you have created at PR #843. Thank you!

Comment (lucasperj) [reply modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase:356]: You can take your own [PR](https://github.com/rodrigocunhaa/liferay-portal/pull/837) as example to what you can do with that one.

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase [hunk 2]
```diff
@@ -315,12 +315,45 @@ definition {
...
+
+		Navigator.gotoNavItem(navItem = "Configuration");
+
+		Workflow.configureWorkflow(
+			workflowDefinition = "Single Approver",
+			workflowResourceValue = "Blogs Entry");
+
+		JSONBlog.addEntry(
+			entryContent = "Blogs Entry Content",
+			entryTitle = "Blogs Entry");
```
Comment (lucasperj) [inline modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase:333]: You can add another one to check the position.

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase [hunk 3]
```diff
@@ -315,12 +315,45 @@ definition {
...
+
+		UserBar.gotoDropdownItem(dropdownItem = "My Workflow Tasks");
+
+		Navigator.gotoNavTab(navTab = "Assigned to My Roles");
+
+		Workflow.updateDueDateTaskByActions(
+			workflowAssetTitle = "Blogs Entry",
+			workflowAssetType = "Blogs Entry",
+			workflowDueDateTime = "1:45 PM",
+			workflowTask = "Review");
```
Comment (lucasperj) [inline modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase:343]: In this piece, you can create and them update the due date of Blogs Entry 2 to a date closer, and Blogs Entry 1 to a later date, then, assign both to you and activate the Due Date filter, use a macro to ensure the expected position.

**Review by lucasperj — CHANGES_REQUESTED** [review]

**Review by lucasperj — APPROVED** [review]

**Comment by lucasperj** [comment]
Just started reviewing :)

**Comment by lucasperj** [comment]
Combined and sent on [PR#3](https://github.com/lucasperj/liferay-portal/pull/3). Thank you!

**Pull request closed without merging**